### DISCUSSION TITLE
Allow hiding the user part of the device name or update based on current user

### DISCUSSION THREAD

**Discussion by andrewheberle** [rustdesk/rustdesk-server-pro#280] [Ideas]
When a RustDesk client is first added the device is named username@device_name however in an environment where different users may log onto a machine this information is no longer correct.

It may be possible to implement this in a few ways if it is useful:

- Show the device name the same as it is now
- Update the username based on the currently logged in user, with only the device name shown if no user is logged in
- Only show the device name
- Allow some configuration option to select the above...maybe via a Strategy?

I had originally added this as a discussion item in the rustdesk/rustdesk repo but this is probably a more appropriate location.

**Comment by rustdesk** [comment]
close and lock to favor https://github.com/rustdesk/rustdesk-server-pro/issues/284#issuecomment-2213579683
